app/views/users/_form.html.haml

```diff
@@ -33,3 +33,4 @@
         = link_to t("form.button.user.deactivate"), deactivate_user_path, class: "govuk-button govuk-button--secondary"
       - else
         = link_to t("form.button.user.reactivate"), reactivate_user_path, class: "govuk-button govuk-button--secondary"
+        = link_to t("form.button.user.anonymise"), anonymise_user_path, class: "govuk-button govuk-button--warning"
```

Comment:
From the card, it seems this should only appear for deactivated users.

Reply:
Completely missed the if here because above the fold. Sorry @benshimmin, resolving
<img width="524" alt="image" src="https://github.com/user-attachments/assets/7d7564f6-ed96-42a0-8c82-d297ca57a973" />